### ISSUE TITLE
1.20 Support

### ISSUE THREAD

**Issue by DarkKingKaan** [GeorgeV220/VoteRewards#130]
The plugin seems to break on versions above 1.19.4.
Would be awesome if the new version support was added.

`[20:28:36] [Server thread/INFO]: [VoteRewards] Enabling VoteRewards v8.3.0
[20:28:36] [Server thread/ERROR]: Error occurred while enabling VoteRewards v8.3.0 (Is it up to date?)
java.lang.NoSuchMethodError: org.yaml.snakeyaml.representer.Representer: method 'void <init>()' not found
	at com.georgev22.library.yaml.file.YamlRepresenter.<init>(YamlRepresenter.java:13) ~[VoteRewards.jar:?]
	at com.georgev22.library.yaml.file.YamlConfiguration.<init>(YamlConfiguration.java:29) ~[VoteRewards.jar:?]
	at com.georgev22.library.yaml.file.YamlConfiguration.loadConfiguration(YamlConfiguration.java:48) ~[VoteRewards.jar:?]
	at com.georgev22.library.yaml.configmanager.CFG.reloadFile(CFG.java:167) ~[VoteRewards.jar:?]
	at com.georgev22.library.yaml.configmanager.CFG.setup(CFG.java:85) ~[VoteRewards.jar:?]
	at com.georgev22.library.yaml.configmanager.CFG.<init>(CFG.java:54) ~[VoteRewards.jar:?]
	at com.georgev22.voterewards.utilities.configmanager.FileManager.loadFiles(FileManager.java:30) ~[VoteRewards.jar:?]
	at com.georgev22.voterewards.VoteReward.onEnable(VoteReward.java:150) ~[VoteRewards.jar:?]
	at com.georgev22.voterewards.votereward.VoteRewardPlugin.onEnable(VoteRewardPlugin.java:52) ~[VoteRewards.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:281) ~[paper-api-1.20.1-R0.1-SNAPSHOT.jar:?]
	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:189) ~[paper-1.20.1.jar:git-Paper-62]
	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104) ~[paper-1.20.1.jar:git-Paper-62]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507) ~[paper-api-1.20.1-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugin(CraftServer.java:636) ~[paper-1.20.1.jar:git-Paper-62]
	at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugins(CraftServer.java:547) ~[paper-1.20.1.jar:git-Paper-62]
	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:636) ~[paper-1.20.1.jar:git-Paper-62]
	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:435) ~[paper-1.20.1.jar:git-Paper-62]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:308) ~[paper-1.20.1.jar:git-Paper-62]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1101) ~[paper-1.20.1.jar:git-Paper-62]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:318) ~[paper-1.20.1.jar:git-Paper-62]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
[20:28:36] [Server thread/INFO]: [VoteRewards] Disabling VoteRewards v8.3.0`

**Comment by GeorgeV220** [comment]
Hey, please try the version 9.0.0
https://github.com/GeorgeV220/VoteRewards/releases/tag/v9.0.0
Player data from version 8.x is no longer supported so it is necessary to create a backup using the command "voterewards backup", remove the old data, install the new version, and finally perform a restoration using the command "voterewards restore ".

**Comment by DarkKingKaan** [comment]
The "VoteRewards Backup" command doesn't exist in the old version. How am I supposed to back up the data before upgrading?

**Comment by GeorgeV220** [comment]
The voterewards backup exists in voterewards from version 4.7.0
https://github.com/GeorgeV220/VoteRewards/blob/92497f38bbc3c7cf5e781f806fcecefac182b8dc/src/main/java/com/georgev22/voterewards/commands/VoteRewardsCommand.java#L156
check if you have the voterewards command enabled in your config.yml

**Comment by DarkKingKaan** [comment]
I see, I had the command disabled.
When I tried making a backup, there was just one message displayed in console that said "100". There was a file created including only my username and data. Perhaps I should have waited a bit longer? We have quite a large database.

I didn't really need any player data so I just deleted the database and let it generate again. After updating the plugin everything works as intended. Thanks for the update!
